test/.eslintrc

```diff
@@ -7,5 +7,8 @@
     "Parsimmon": true,
     "assert": true,
     "testSetScenario": true
+  },
+  "rules": {
+    "no-invalid-regexp": ["error", {"allowConstructorFlags": ["u", "y"]}]
```

Comment:
ESLint doesn't like these flags. And this override doesn't work if you define the Regexp like `/a/gimuy`, you have to use the `new Regexp` constructor, apparently.